docs/index.html

```diff
@@ -148,6 +148,36 @@ <h2>Installation</h2>
         There exist no drivers for MacOS, currently.
       </p>
 
+      <p>
+        FreeBSD supports <em>Noted</em> layout whether you are using X or the console (VT or syscons). Using X, 
+        the Linux instructions mostly applies.
+      </p>
+      <p>
+        Regarding usage on FreeBSD's console, following is an excerpt from 
+       <a href="https://www.neo-layout.org/Einrichtung/BSD/">Setup Neo > BSD Systems</a> (be mindful to replace 
+       any mentions of neo.kbd with 
+       <a href="https://github.com/Jaso-N7/noted-layout/blob/main/kbd/noted.kbd">noted.kbd.</a>).
+       <br>
+
+        As mentioned in the <strong>NOTE</strong>: <em>This driver is  not intended as a complete replacement, however 
+       it will make basic system administration possible using level 1–3 without switching to X.</em>
+      </p>
+      <p>
+        For FreeBSD on the console, the <a href="https://github.com/Jaso-N7/noted-layout/blob/main/kbd/noted.kbd">noted.kbd</a> 
```

Comment:
This link should be pointing to the location after the merge, i.e. `https://github.com/dariogoetz/noted-layout/blob/main/kbd/noted.kbd`